src/lib/octokit.js

```diff
@@ -39,3 +43,9 @@ module.exports = token => {
 
 	return client
 }
+
+module.exports.clearPendingMessages = () => {
+	labels.forEach(label => {
+		logger.log(label, { status: 'done' })
+	})
```

Comment:
**issue**: this should also clear the array, otherwise, if it's called again, it'll log things that have already cleared